### PR TITLE
41,38 change sommore things and remove school secton from nav

### PR DESCRIPTION
**Pull request by faizanxyz**

### CODE REVIEW

File: src/components/Navbar/components/howItWorks.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const HowItWorks: React.FC<IProps> = ({ navType }) => {
   return (
     <>
       <Box bg={navType == 'primary' ? primary.colors.ED.white : primary.colors.ED.primary} w='100%' borderRadius='6'>
-        <VStack p={20}>
+        <VStack p={"25px 41px 63px 41px" }>
```
Comment (usama-maxenius) [inline src/components/Navbar/components/howItWorks.tsx:26]: It should be {['25px', '41px' , '63px', '41px']" }

File: src/components/Navbar/components/whyED.tsx [hunk 2]
```diff
@@ -20,7 +20,7 @@ const WhyED: FC<IProps> = ({navType}) : JSX.Element => {
           <Box w='100%' p={6} bg={navType == 'primary' ? primary.colors.ED.white : primary.colors.ED.primary} className='primaryContainer' borderRadius='6'>
             <Flex gap='5'>
               <Box w='50%' display='flex' alignItems='center' flexDir='column' gap={4}>
-                <Text fontFamily='IBM Plex Sans' fontWeight='500' fontSize='4xl' color={navType == 'primary' ? primary.colors.ED.primary : primary.colors.ED.white}>
+                <Text fontFamily='IBM Plex Sans' fontWeight='500' fontSize='32px' color={navType == 'primary' ? primary.colors.ED.primary : primary.colors.ED.white}>
```
Comment (usama-maxenius) [inline src/components/Navbar/components/whyED.tsx:23]: Please specify font size in an array like my previous comments for every screen 

{'Mobile', 'Tablet' , 'Small Laptop','1920 and bigger '}

File: src/components/NewNav/index.tsx [hunk 3]
```diff
@@ -105,17 +105,17 @@ const Navbar: FC = (): JSX.Element => {
...
+                                                {/* <Popover placement='bottom-end' trigger='hover' arrowSize={30}
                                                     arrowShadowColor={navType == 'primary' ? primary.colors.ED.white : primary.colors.ED.primary}
                                                 >
                                                     <PopoverTrigger>
                                                         <Text role='button'>Schools</Text>
                                                     </PopoverTrigger>
                                                     <PopoverContent inset={navType == 'primary' ? '35px 0px auto 32px' : '20px 0px auto 32px'} w='calc(100% - 30px)' m='0 auto' border='none' boxShadow='xl' _focusVisible={{ outline: 'none' }}>
-                                                        <PopoverArrow bg={navType == 'primary' ? primary.colors.ED.white : primary.colors.ED.primary} zIndex={1} />
+                                                        <PopoverArrow bg={navType == 'primary' ? primary.colors.ED.white : primary.colors.ED.primary} zIndex={1} marginLeft={'-30px'}/>
```
Comment (usama-maxenius) [inline src/components/NewNav/index.tsx:115]: Why we are giving margin in negative?

**Review by usama-maxenius — CHANGES_REQUESTED** [review]
Please fix the issues for which I added the comments

File: src/components/Navbar/components/howItWorks.tsx (hunk 1 above)
Comment (faizanxyz) [reply src/components/Navbar/components/howItWorks.tsx:26]: usama bhai ya padding {top right bottom left } is way sa de ha

File: src/components/NewNav/index.tsx (hunk 3 above)
Comment (faizanxyz) [reply src/components/NewNav/index.tsx:115]: bhai ya portion comment karwa diya tha apna

**Pull request merged**